### ISSUE TITLE
It's Clipboard read & write allowed?

### ISSUE THREAD

**Issue by rullymartanto** [robbestad/react-iframe#125]
I need to add clipboard read and write on iframe, it's possible?

**Comment by BenGedi** [comment]
+1

**Comment by robbestad** [comment]
Yes — this is Permissions-Policy, not a library flag: `allow="clipboard-read; clipboard-write"`. The framed document must also be allowed to use the Clipboard API. Documented in the 2.0 FAQ.
